scripts/download_wordnet.py

```diff
@@ -0,0 +1,3 @@
+import nltk
```

Comment:
what is the `nltk`?

Reply:
https://pythonprogramming.net/wordnet-nltk-tutorial/

> WordNet is a lexical database for the English language, which was created by Princeton, and is part of the NLTK corpus.